.dockerignore

```
@@ -2,6 +2,9 @@
 .github/
 .git/
 
+node_modules
```

the other directories end with `/` (probably good practice to prevent the possibility of matching a file with the same name). If this is like `.gitignore` it will also ignore sub-directories with the same name unless prefixed with `/`. I wonder if we should prefix everything in this file with `/` or remove the other instances of `node_modules` that are in sub-directories?
```suggestion
node_modules/
```